### DISCUSSION TITLE
Change message.success icon color doesn't work with ConfigProvider

### DISCUSSION THREAD

**Discussion by billbrinck** [ant-design/ant-design#39855] [Q&A]
I am currently trying to change the theme of antd. However I am unable to change the icon color for messages:

![wJcNP](https://user-images.githubusercontent.com/22788111/209667554-6f137046-45d8-4afc-90f4-17deb34fcea2.png)

This is what I currently have:

    <ConfigProvider
        theme={{
          token: {
            yellow: colors.yellow500,
            blue: colors.blue500,
            green: colors.teal500,
            red: colors.red400,
            colorPrimary: colors.blue500,
            colorError: colors.red400,
            colorInfo: colors.blue500,
            colorSuccess: colors.teal500,
            colorWarning: colors.yellow500,
            lime: colors.lime500,
            purple: colors.purple500,
            cyan: colors.cyan500,
            colorLink: colors.blue500,
          }
        }}
      >

Seems that with the ConfigProvider it is not working.

The icon color should match the successColor, configured via the ConfigProvider's theme parameter.
I also want to change the message.error, message.warning and message.info color later on.

**Comment by uraden** [comment]
Hi, were you able to find the solution?
